### PR TITLE
168 improve test coverage if necessary

### PR DESCRIPTION
**Pull request by Ben-Rey**
This is very arbitrary as it would require more analysis to know what we need to test.
The main purpose of this pr is to exceed the 70% coverage.

### CODE REVIEW

File: assembly/std/__tests__/context.spec.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  transferredCoins,
+} from '../context';
+
+describe('Context', () => {
+  test('test ownedAddresses', () => {
+    const addresses: Address[] = ownedAddresses();
+    expect(addresses.length).toBeGreaterThan(0);
+  });
+
+  test('test caller', () => {
```
Comment (gregLibert) [inline assembly/std/__tests__/context.spec.ts:16]:
```suggestion
  test('caller', () => {
```

File: vm-mock/vm.js [hunk 2]
```diff
@@ -364,6 +364,14 @@ export default function createMockedABI(
 
         return newArrayBuffer(keys);
       },
+
+      assembly_script_get_owned_addresses() {
+        return newString(`[ ${callerAddress} , ${contractAddress} ]`);
```
Comment (gregLibert) [inline vm-mock/vm.js:369]: What is ownedAddresses supposed to return ?

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: vm-mock/vm.js (hunk 2 above)
Comment (Ben-Rey) [reply vm-mock/vm.js:369]: It's not documented yet, I just deduced that it was an address array in json format: 
/**
 * Returns owned addresses.
 *
 * TODO:
 * - explain function purpose
 * - explain format
 *
 */
 
 So for now I return an array of address in json format. I plan to improve the comment in the documentation issue

**Review by gregLibert — APPROVED** [review]

File: as-pect.config.js [hunk 3]
```diff
@@ -15,7 +16,7 @@ export default {
   },
 
   // Enable code coverage.
-  coverage: ['assembly/**/*.ts'],
+  // coverage: ['assembly/**/*.ts'],
```
Comment (Thykof) [inline as-pect.config.js:19]: please remove commented out code

**Review by Thykof — CHANGES_REQUESTED** [review]

File: as-pect.config.js (hunk 3 above)
Comment (Ben-Rey) [reply as-pect.config.js]: Done

File: as-pect-ci.config.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import config from './as-pect.config.js';
+config.coverage = ['assembly/**/*.ts'];
+export default config;
```
[the remark this answers is not in the document]
Comment (Ben-Rey) [reply as-pect-ci.config.js:3]: I added a custom configuration to enable coverage only for the CI, but there may be a better way.

**Pull request merged**